Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -2077,25 +2077,25 @@ void ACesium3DTileset::Tick(float DeltaTime) {
         CreateViewStateFromViewParameters(camera, unrealWorldToCesiumTileset));
   }
 
-  Cesium3DTilesSelection::ViewUpdateResult result;
+  const Cesium3DTilesSelection::ViewUpdateResult* result;
```

Comment:
Nitpick, but it would be better to name this `pResult`. https://github.com/CesiumGS/cesium-native/blob/main/doc/style-guide.md#-naming

Sometimes we use Unreal's naming conventions instead (we should be more consistent about this), in which case the proper name would be `Result`.

I'll make this change and then merge this in order to avoid another cycle just for trivial changes.